gradle.properties

```diff
@@ -0,0 +1,3 @@
+slf4jVersion=2.0.17
```

Comment:
nitpick (non-blocking): I wonder how dependabot plays out with it like this, I had avoided it by defining it in the build.gradle itself (it's a curiousity).